mesonbuild/ast/printer.py

```diff
@@ -5,6 +5,8 @@
 # or an interpreter-based tool
 from __future__ import annotations
 
+from attr import has
```

Comment:
Where does this come from??

Reply:
... I deleted that so many times. My editor seems to think I need that for some reason I can't fathom